Samples/Samples/View/CalendarEventPage.xaml

```diff
@@ -62,9 +62,36 @@
                     <ColumnDefinition Width="Auto"/>
                     <ColumnDefinition/>
                 </Grid.ColumnDefinitions>
-                <Label Text="Description:"/>
-                <Label Grid.Column="1"
-                   Text="{Binding Description}"/>
+              <Grid.RowDefinitions>
+                  <RowDefinition/>
+                  <RowDefinition/>
+              </Grid.RowDefinitions>
+                <Label Text="Attendees"/>
```

Comment:
Should this have : to be consistent with Description label

Reply:
Yep, I believe I altered this in a future commit, but have adjusted for this PR too!